### ISSUE TITLE
Hi Evert

### ISSUE THREAD

**Issue by petebannister** [evert/adventofcode#1]
Hows it going? 
Others on the AOC leaderboard:

https://github.com/petebannister/aoc2020

https://github.com/richardgill/advent-of-code-2020

https://github.com/Hephaestuz/AdventOfCode2020

**Comment by evert** [comment]
Thank you! I was curious =D

**Comment by petebannister** [comment]
I see you are doing this again this year!  Also having a go - will see how long I can keep up with it as have a 10 month old baby this time around!

Learning a new language for this one.

https://github.com/petebannister/aoc2021

**Comment by evert** [comment]
@petebannister Ohh congrats!!! 

Good luck! I think I got to 17 or so the last year, but I see you made it all the way to the end. Impressive, I totally lost steam after a while!

**Comment by petebannister** [comment]
Yes took dedication!!  Some of them started taking way too long.  
Marriage already starting to suffer this time around - not sure how far 
I'll get!

------ Original Message ------
From: "Evert Pot" ***@***.***>
To: "evert/adventofcode" ***@***.***>
Cc: "petebannister" ***@***.***>; "Mention" 
***@***.***>
Sent: 03/12/2021 00:38:05
Subject: Re: [evert/adventofcode] Hi Evert (#1)

>
***@***.*** <https://github.com/petebannister> Ohh congrats!!!
